log4j-1.2-api/src/test/java/org/apache/log4j/CategoryTest.java

```diff
@@ -98,35 +98,35 @@ public void testForcedLog() {
         category.info("Hello, World");
         List<LogEvent> list = appender.getEvents();
         int events = list.size();
-        assertTrue("Number of events should be 1, was " + events, events == 1);
+        assertTrue(events == 1, "Number of events should be 1, was " + events);
```

Comment:
```suggestion
        assertEquals(1, events, "Number of events");
```

A couple of old assertions did use `assertTrue` to test the equality of two values. Could your replace them with `assertEquals` please?

Reply:
Thanks